src/measure_theory/measure_space.lean

```diff
@@ -1444,8 +1436,37 @@ instance probability_measure.to_finite_measure (μ : measure α) [probability_me
 lemma probability_measure.ne_zero (μ : measure α) [probability_measure μ] : μ ≠ 0 :=
 mt measure_univ_eq_zero.2 $ by simp [measure_univ]
 
+lemma prob_add_compl_eq_one [probability_measure μ]
+  (h: measurable_set s) : μ s + μ sᶜ = 1 :=
+begin
+  rw measure_add_measure_compl,
+  rw measure_univ,
+  exact h,
+end
+
+lemma prob_le_one [probability_measure μ] (measurable_set s) : μ s ≤ 1 :=
```

Comment:
Usually an "unused hypothesis" linter warning means that you should just remove the entire hypothesis like this:
```suggestion
lemma prob_le_one [probability_measure μ] : μ s ≤ 1 :=
```

Reply:
So the "s" does not need to be declared? I do not understand. I see that two instances probability_measure and measurable_space show up.